resources/views/emails/messages/plain_text.blade.php

```diff
@@ -0,0 +1,6 @@
+Dear {!! isset($recipientName) ? $recipientName : $recipient->name !!},
+
+{{  strip_tags($body) }}
```

Comment:
Blade `{{ }}` blocks will be run though `htmlspecialchars`, which should make `strip_tags` redundant